artiq/frontend/artiq_mkfs.py

```diff
@@ -41,6 +64,22 @@ def main():
         for key, filename in args.f:
             with open(filename, "rb") as fi:
                 write_record(fo, key, fi.read())
+        for action in args.e:
+            if action == "write_ch_names":
+                ddb = DeviceDB(args.device_db).get_device_db()
+                channel_ntn = channel_number_to_name(ddb)
+                if not channel_ntn:
+                    print("No device with channel number is found in device database")
+                else:
+                    channel_names = []
+                    print("Write:")
```

Comment:
That's unnecessarily verbose. You should use debug logging instead.

And anyway this becomes completely unnecessary if you break out this device_db > config string into a separate frontend tool that does only this one thing and is trivial to debug.

Please also update the manual and release notes.